.github/workflows/fossa.yaml

```diff
@@ -0,0 +1,29 @@
+name: Fossa scan
+
+on:
+    push: 
+      branches:
+        - '*'
```

Comment:
```suggestion
        - 'release/v*'
```

Reply:
updated to '**' which should cover all branches